scripts/build_components.sh

```diff
@@ -96,9 +104,21 @@ for dir in "${components_to_build[@]}"; do
   echo "Freezing Fondant dependency version to ${tags[0]}"
   docker build --push "${args[@]}" \
    --build-arg="FONDANT_VERSION=${tags[0]}" \
-   --label org.opencontainers.image.source=https://github.com/${namespace}/${repo} \
+   --label org.opencontainers.image.source=https://github.com/${repo}/components/{BASENAME} \
```

Comment:
I don't think we still need this

Reply:
No, but it's still useful metadata I think, so I left it in.

Reply:
ok makes sense